### DISCUSSION TITLE
打包docker容器出现依赖安装问题

### DISCUSSION THREAD

**Discussion by KangEn1994** [mikumifa/biliTickerBuy#313] [提问专区]
bili-ticket-gt-python  这个依赖无法安装，尝试注释掉打包成功
启动显示这个是本地验证码模块，这个是在升级么

**Comment by mikumifa** [comment]
有的系统不支持bili-ticket-gt-python

**Comment by mikumifa** [comment]
但是按道理docker应该也支持啊

**Reply by KangEn1994** [reply]
很怪。。。尝试用两台机器进行了打包，A机器失败了，B机器成功了
